changelog.md

```diff
@@ -1,3 +1,13 @@
+2.17.1
+=========
+- Updated to Microsoft.IdentityModel.* 7.4.0
+
+### New features
+- Calls to `ConfidentialClientApplicationBuilderExtension.WithClientCredentials` are fully async. See issue [#2566](https://github.com/AzureAD/microsoft-identity-web/issues/2566) for details.
```

Comment:
I think it's more engineering excellence. This won't change the public API?

Reply:
done. thanks